tests/foreman/api/test_discoveredhost.py

```diff
@@ -353,29 +353,32 @@ def test_positive_refresh_facts_pxe_host(self, module_target_sat):
         """
 
     @pytest.mark.on_premises_provisioning
+    @pytest.mark.parametrize('endpoint', ['reboot', 'reboot_all'])
```

Comment:
Instead of making it parametrized, I'd suggest to keep it as a separate test

Reply:
Any specific reason for separate test, both would do the same things?

Reply:
Like if first endpoint fails for some reason, then second endpoint will be untested, so I'd suggest to keep as it is in separate tests

Reply:
Updated, it was causing some issues. Separate two tests